marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/job/findStepResponses.api

```diff
@@ -0,0 +1,20 @@
+{
+  "endpoint": "/data-hub/5/data-services/stepResponses/findStepResponses.sjs",
```

Comment:
I'm surprised this works, but the endpoint should be "data-services/job" instead of "data-services/stepResponses", right?